Stripe/PublicHeaders/STPPaymentConfiguration.h

```diff
@@ -33,42 +34,75 @@ NS_ASSUME_NONNULL_BEGIN
 /**
  *  Your Stripe publishable key. You can get this from https://dashboard.stripe.com/account/apikeys .
  */
-@property(nonatomic, copy)NSString *publishableKey;
+@property (nonatomic, copy) NSString *publishableKey;
 
 /**
- *  An enum value representing which payment methods you will accept from your user in addition to credit cards. Unless you have a very specific reason not to, you should leave this at the default, `STPPaymentMethodTypeAll`.
+ *  If YES, STPPaymentContext will generate STPSource objects when creating
+ *  new credit cards. Otherwise it will generate STPCard objects.
+ *
+ *  The default value is NO
+ *
+ *  @see https://stripe.com/docs/sources
  */
-@property(nonatomic)STPPaymentMethodType additionalPaymentMethods;
+@property (nonatomic) BOOL useSourcesForCreditCards;
 
 /**
- *  The billing address fields the user must fill out when prompted for their payment details. These fields will all be present on the returned token from Stripe. See https://stripe.com/docs/api#create_card_token for more information.
+ *  An ordered set of payment method type objects that represents the list of
+ *  the available payment methods available to users of your app.
+ * 
+ *  The methods in this list will be shown to users in STPPaymentMethodsViewController
+ *  for users to choose from in the order they are listed in this array.
+ *
+ *  Any methods types not in either this array will not be available to users 
+ *  of your app.
+ *
  */
-@property(nonatomic)STPBillingAddressFields requiredBillingAddressFields;
+@property (nonatomic, copy) NSOrderedSet<STPPaymentMethodType *> *availablePaymentMethodTypes;
```

Comment:
Can we make this an NSArray externally and an NSOrderedSet internally (for better swift compatibility)?